jdbc/mysql-j-5/pom.xml

```diff
@@ -133,7 +133,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <scope>test</scope>
+      <scope>compile</scope>
```

Comment:
does it need to be compile or should we just leave it to the default (which I think is compile)?

Reply:
I think default scope should be fine.